src/main/resources/languages/messages_en.yml

```diff
@@ -152,6 +152,8 @@ messages:
     whirlwind: '&a&oYour Talisman reflected the Projectile'
     wizard: '&a&oYour Talisman has given you a better Fortune Level but maybe also lowered some other Enchantment Levels'
     caveman: '&a&oYour Talisman gave you haste'
+    wise: '&a&oYour Talisman doubled your experience drops'
+    resurrected: '&a&oYour Talisman a-voided your death'
```

Comment:
puns  👍 But this is consumable right? So you may wanna add to it that the talisman was consumed

Reply:
Ill make another PR somewhat soon to fix the other talisman messages, they have capitals all over the place and no mentions of being consumable